contracts/ExtraRewardStash.sol

```diff
@@ -1,13 +1,17 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.14;
+pragma solidity 0.8.15;
 
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "./utils/Interfaces.sol";
+import "./utils/MathUtil.sol";
 
 /// @title ExtraRewardStash
 contract ExtraRewardStash is IStash {
```

Comment:
Do this fixes `S1`?
I think we haven't added a function for controller to remove token?

Reply:
I've added function to clear rewards on this contract as well for the multisig